src/components/Menu/Menu.tsx

```diff
@@ -13,23 +14,21 @@ import "./Menu.css";
 
 export type MenuProps = Omit<ReactMenuProps, "menuButton" | "theming" | "transition"> & {
   isDisabled?: boolean;
+  menuButton?: ReactMenuProps["menuButton"];
 };
```

Comment:
```suggestion
export type MenuProps = Omit<ReactMenuProps, "theming" | "transition"> & {
  isDisabled?: boolean;
};
```

Maybe I am missing something, but can we just remove `menuButton` from omitted props instead of adding it again?